lib/eventsource/events.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { webidl } = require('../fetch/webidl')
+const {
+  MessageEvent,
+  ErrorEvent,
+  eventInit
+} = require('../websocket/events')
+
+class OpenEvent extends Event {
```

Comment:
OpenEvent could be moved to websocket/events.js as Websocket is also having an OpenEvent. wdyt?

Reply:
what's the difference between this one and WebSocket's?

Reply:
Nothing they are basically just Events. I just made a dedicated class for it.

Reply:
I haven't taken a look at the spec yet, but there's a difference between "an event named 'open'" and an OpenEvent. If WebSocket doesn't have its own OpenEvent, it's likely just `new Event('open')` that's required